docs/contributing/developing.rst

```diff
@@ -116,6 +116,35 @@ If your Elasticsearch instance is located somewhere else, you can set the
 
     $ ELASTICSEARCH_URL=http://my-elasticsearch-instance:9200 python runtests.py --elasticsearch
 
+**Browser and device support**
+
+Wagtail is meant to be used on a wide variety of devices and browsers. Supported browser / device versions include:
+
+=============  =============  =============
+Browser        Device/OS      Version(s)
+=============  =============  =============
+Mobile Safari  iOS Phone      Latest
+Mobile Safari  iOS Tablet     Latest
+Chrome         Android        Latest
+IE             Desktop        9, 10, 11
+Chrome         Desktop        Latest
+MS Edge        Desktop        Latest
+Firefox        Desktop        Latest
+Safari         macOS          Latest
+=============  =============  =============
+
+Those environments are the ones Wagtail *is tested* on. Our development standards ensure that the site is usable on other browsers **and will work on future browsers**. To test on IE, install virtual machines `made available by Microsoft <https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/>`_.
```

Comment:
I think we should come up with clear testing guidelines here, balance our wish to support as many people as possible with the diminishing returns of browser support. 

Personally I don't think there is much of a point in stating support if there are no tests, but that's a lot of tests to ask for external contributors. Perhaps we could leverage something like [Sauce Labs](https://saucelabs.com/) or [BrowserStack](https://www.browserstack.com/) (both being free for open-source). How has Wagtail handled browser tests / browser support so far?